README.md

```diff
@@ -1,70 +1,152 @@
 
-#### Reviewer
+For documentation on the commands found in these files please see the [Django documentation](https://docs.djangoproject.com/en/2.1/).
 
-When someone requests you review their work, make sure you get a copy of that person's pull request and test their code locally. To get the pull request as a new branch, follow [these steps](https://help.github.com/articles/checking-out-pull-requests-locally/#modifying-an-inactive-pull-request-locally)
+## Key Terms:
+* Workflows - A visual representation of a script's main functions that shows the flow of data throughout the program.
+* Versions - A distinctly different version of a workflow. Versions are used to distinguish when a workflow has been updated in any way.
+* Runs - A run through of data inputs on a specific version of a specific workflow. Runs show inputs and outputs into a script and help give a better understanding of where data is being handled and manipulated
+* Tags - optional identifying keywords
 
-Summary:
+## CLI Tool
+### YesWorkflow Save Command and Configurations
+Note: To use the jar file please install the following prequisite:
+* Java (JRE) version 1.10
 
-Use `git fetch origin pull/<Merge Number>/head:<Name of new Branch>`, then checkout the newly created branch
+The Jar file will need to be executed from the terminal and can be found at the root folder of the YesWorkflow Web Components repository.
 
+YesWorkflow CLI's Save command works by executing and aggregating all YesWorkflow commands and then sending important data up to your deployed YesWorkflow WebComponents instance. To use, type in the command prompt below
 
-## Setting up YesWorkflow dev Environment
+### Example usage:
+In this example my_scripty.py is a python file that contains YesWorkflow annotations. The commands will work the same on any .matlab or .R files that contain YesWorkflow annotations.
 
-Most recent version of [YesWorkflow CLI Java implementation](https://github.com/aniehuser/yw-prototypes)
+```
+java -jar yesworkflow.jar save my_scripty.py
+```
 
-YesWorkflow is a maven project. To set up the project for development follow these steps:
+You can set configurations manually in the command prompt
+```
+java -jar yesworkflow.jar save my_scripty.py -c save.serveraddress=http://localhost:8000/
+```
 
-1. Ensure you have a Java SE 7 or above installed on your device. To see all versions on a mac device and their location, run `$ /usr/libexec/java_home -V`.
-2. Set JAVA_HOME to any version that is 1.7 or greater. I.e. on mac `$ export JAVA_HOME="/usr/libexec/java_home -v 1.8"` for Java 8.
-3. Ensure [Maven 3.2](https://maven.apache.org/install.html) or above is installed on your device.
-4. Run `$ git clone https://github.com/aniehuser/yw-prototypes.git`.
-5. Navigate to the newly cloned directory.
-6. Open the `pom.xml` file in the project's root directory and edit the following portion to reflect the current java and maven version
```

Comment:
same here, if we're going to remove these instructions, lets move them to another file. I don't think we want to just delete this.